### ISSUE TITLE
Game Complete - Declare Winner

### ISSUE THREAD

**Issue by jasonye85** [willowtreeapps/wombats-web-client#30]
As a user, I would like to view the winner of a game after it is played. 

***AC:***
* Display winner, Wombat image, Wombat name, github handle 
* Return to Games screen

***Zeplin:***
https://zpl.io/1hcgHT

***Out of Scope:***
Holding off on replay button for V2

**Comment by morganedmonds** [comment]
The API does not currently notify of end of game, so this ticket may require API work.

**Comment by morganedmonds** [comment]
Blocked by API, need game state changes sent through sockets

**Comment by emilyseibert** [comment]
PR #134 is covering styles for modal, need to implement integration logic

**Comment by morganedmonds** [comment]
This excludes styling and the case of a tie; moving to design review for Jesse to take a look at on Friday.
